### DISCUSSION TITLE
Control MacBook Screen Brightness as well

### DISCUSSION THREAD

**Discussion by LinusHe** [MonitorControl/MonitorControl#1151] [General]
### Before opening the issue, have you...?

- [X] Searched for existing issues

### Is your feature request related to a problem? Please describe

Is it possible to implement a slider to control the internal macbook screen brightness in the app as well?

### Describe the solution you'd like

I've seen that you can add multiple external screens in the app to control them in the menu bar popup. It would be cool to also have an option there to control the internal screen brightness.

### Describe alternatives you've considered

Right now I just use the keyboard to adjust the macbooks internal screen brightness, but I've issues with that. Sometimes the shortcuts keys just adjust the external and sometimes just the internal screen brightness. I have noticed, that it depends on where you're mouse focus was at the last time.

**Comment by waydabber** [comment]
The app should already control the internal screen brightness (it is usually under `Color LCD`). Everything should work the same way as with the external displays.

<img width="274" alt="Screenshot 2022-08-26 at 17 53 49" src="https://user-images.githubusercontent.com/37590873/186944763-f556884e-0027-4e3e-bc67-07dbad840236.png">

**Comment by LinusHe** [comment]
@waydabber Thanks a lot for your reply.
Sadly I don't have that option: 
![image](https://user-images.githubusercontent.com/26716809/187043658-0f8e3af7-d688-45ad-a328-7460ee2c4aaa.png)

**Reply by waydabber** [reply]
Hmm. This does not look like an up-to-date version of MonitorControl.
